content/articles/creating-a-simple-chat-app-in-react-using-the-chat-engine-api/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Creating a Simple Chat App in React.js using the chat engine API
+---
+
+This article will teach the React.js way of creating a chat application using the chat engine API. It aims at making it less cumbersome compared to when one opts the Node.js and socket.io way instead.
+
```

Comment:
This first sentence on you introduction is quite confusing. Why are you comparing Node.js and Socket.io? What if readers have no prior knowledge?